lib/trivial.nix

```
@@ -354,14 +354,14 @@ rec {
 
     Type: bool -> string -> a -> a
   */
-  warnIf = cond: msg: if cond then warn msg else x: x;
+  warnIf = cond: msg: if cond then lib.warn msg else x: x;
```

```suggestion
  warnIf = cond: msg: if cond then warn msg else x: x;
```

Please bring the functions we need into scope with `let inherit` so that we save an attribute lookup, which could be relevant in some code. It also keeps the git blame a bit cleaner.
(Applies to all `lib.` changes in this file)